app/tournamentBusiness/tournamentProgress.js

```diff
@@ -0,0 +1,121 @@
+import $ from "jquery";
+import * as tournamentAlerts from "./tournamentAlerts";
+
+var tournament;
+
+export function init(t) {
+    tournament = t;
+
+    generateHTMLTournamentBracket();
+}
+
+function generateHTMLTournamentBracket() {
+    cleanBracket();
+    divideBracketVerticallyInRounds(tournament.getRounds());
+    appendAllMatchesWithSpacesToHtml(tournament.matches);
+    bindClickEventOnWinButtons(tournament.matches);
+}
+
+function cleanBracket() {
+    $("#tournamentBracket").empty();
+}
+
+function divideBracketVerticallyInRounds(rounds) {
+    var colSize = 100/rounds;
+
+    for (var r = 1; r <= rounds; r++) {
+        $("#tournamentBracket").append(
+            `<div class="col-xs-1" id="round${r}" style="width: ${colSize}%; padding: 0;"></div>`
+        );
+    }
+}
+
+function appendAllMatchesWithSpacesToHtml() {
+    tournament.matches.forEach( function(match, i, matches) {
+        if (match.round > 1) {
+            appendTransparenSpacesToHtml(match.round, matches[i-1].round);
+        }
+        appendMatchHtmlPanel(match, i);
+    });
+}
+
+function appendTransparenSpacesToHtml(round, lastRound) {
+    $(`#round${round}`).append(
+        generateHtmlTransparentSpaces(round, lastRound)
+    );
+}
+
+function generateHtmlTransparentSpaces(round, lastRound) {
+    var space = "";
+
+    if (round !== lastRound) {
+        if (round > 1) {
+            space += `<div class="panel panel-default box-bracket-ghost-med"></div>`;
+        }
+        if (round > 2) {
+            space += `<div class="panel panel-default box-bracket-ghost"></div>`.repeat(round - 2);
```

Comment:
This approach is original but hard to validate. Have you considered an HTML table? No need to revamp, just think on it.